tests/test_public_endpoints.py

```diff
@@ -17,6 +17,7 @@
     slack_connect_redirect_shortcuts.__name__,
     "get_run_status",  # needs query params
     "get_balance",  # needs authentication
+    "internal/file-upload",  # needs admin authentication
```

Comment:
@dev can you check why the tests are failing on /internal/file-upload/ even though I have included it here